pages/common/rdfind.md

```diff
@@ -1,6 +1,6 @@
 # rdfind
 
-> Finds duplicate files and gets rid of them.
+> Find files with duplicate content and gets rid of them.
```

Comment:
There is a minor grammatical error here - It should be either Finds .. and gets, or Find .. and get. Choose one.